packages/sdk-react/src/components/counterparts/CounterpartsTable/CounterpartsTable.tsx

```diff
@@ -355,6 +366,46 @@ const CounterpartsTableBase = ({
     return <AccessRestriction />;
   }
 
+  if (!isLoading && counterparts?.data.length === 0) {
+    return (
+      <DataGridEmptyState
+        icon={
+          <NoDataIcon sx={{ fontSize: '4rem', color: 'primary.main', mb: 2 }} />
```

Comment:
This code setting an icon repeats across every table component. Let's move it to the DataGridEmptyState itself and remove the icon property. There is no need to implement customizability "just in case", until it's not strictly required.